docs/cep/reference/extensions/execution/fx.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Function Worker(Fx)
+---
+
+The `fx` extension executes an Edge Worker.
+
+## Syntax
+
+Executes parameterless function:
+
+    <OBJECT> fx(<STRING> name)
+
+Executes function with parameters set via JSON or object:
+
+    <OBJECT> fx(<STRING> name, <STRING|OBJECT> json)
+
+Executes function with parameters set as pairs of arguments:
+
+    <OBJECT> fx(<STRING> name, <STRING> param1Name, <String|Bool|Int|Long|Double|Float> param1Value, ...)
+
+Examples of function with different syntax:
+```js
+	fx('fn-name')
+    fx('fn-name', \"{name : 'John', age : 23}\")
+    fx('fn-name', json:getObject(\"{name : [{name:'John'}], age : 23}\",'$'))
+    fx('fn-name', 'name', 'John', 'age', 23)
+```
+
+## Example
+
+```js
+@App:name("fx-1")
+@App:description("This app will execute a function every 5 seconds")
+@App:qlVersion('2')
+
+CREATE TRIGGER MyTrigger WITH ( interval = 5 sec );
+
+CREATE SINK STREAM FxSampleStream (result object);
+
+INSERT INTO FxSampleStream
+fx('my-fn-name', 'my-param', 'my-value') as result
+FROM MyTrigger;
+```
+
+For this example needs to be generated Edge Worker with name `my-fn-name`.
+The easiest way is to:
+1. generate Edge Worker based on Query Worker with the next query:
+```sql
+FOR doc IN my-colection LIMIT 1 RETURN doc
+```
+2. create a collection with a name `my-colection` 
+3. add one document into the collection.
+
+Following stream `FxSampleStream` is receiving every 5 seconds result of function worker.
```

Comment:
```suggestion
The following stream `FxSampleStream` is receiving a new message every five seconds as the result of the function worker.
```

Reply:
I am confused. Where did this stream come from? Did we make it in the stream worker you listed above?

Reply:
The `FxSampleStream` stream creates automatically when the stream worker publishes. We do not need to do anything extra.